docs/cccl_development/macro.rst

```diff
@@ -213,12 +213,6 @@ The following macros are required only if the target C++ version does not suppor
 +-----------------------------+----------------------------------------------------------+
 | ``_CCCL_STD_VER``           | C++ standard version, e.g. ``#if _CCCL_STD_VER >= 2017`` |
 +-----------------------------+----------------------------------------------------------+
-| ``_CCCL_IF_CONSTEXPR``      | Portable ``if constexpr`` (before C++17)                 |
-+-----------------------------+----------------------------------------------------------+
-| ``_CCCL_CONSTEXPR_CXX14``   | Enable ``constexpr`` for C++14 or newer                  |
-+-----------------------------+----------------------------------------------------------+
-| ``_CCCL_CONSTEXPR_CXX17``   | Enable ``constexpr`` for C++17 or newer                  |
-+-----------------------------+----------------------------------------------------------+
```

Comment:
Nitpick: those are not removed with the PR but meh

Reply:
Agreed, since you have to rebase anyway, please readd those.

Reply:
Issue for removing those: #3780